### DISCUSSION TITLE
가격 테이블을 어떻게 나눌까 ?

### DISCUSSION THREAD

**Discussion by YurimYang** [SCBJ-7/SCBJ-BE#7] [Ideas]
### 안건
- 현재, 상품의 가격을 어떻게 저장할지에 대해 고민 중이다. 
- 가격이 성수기, 비성수기에 따라 달라지는 것을 고려하고자 한다. 그렇다면, 가격을 객실테이블 안에 칼럼 하나로 관리하기에는 부족하다고 판단했다.

### 고민사항
- 성수기, 비성수기, 주중, 주말을 테이블을 따로 빼서 관리한다.
- 장점 : open api에 데이터가 존재해서 데이터를 쉽게 관리할 수 있다. 
![재철님 ERD](https://github.com/SCBJ-7/SCBJ-BE/assets/63856521/43a91922-eadc-4ac7-bcfd-17a4b9ef061c)

- 날짜별로 가격을 미리 지정을 해서 3개월간의 데이터를 관리한다.
- 장점 : 미리 데이터를 갖고 있을 수 있다. 
- 단점 : 이전 데이터를 어느시점에 삭제해야할지 의문이다. 
![image](https://github.com/SCBJ-7/SCBJ-BE/assets/63856521/98202fb1-0006-45b9-8dc3-e9542523268d)

**Comment by Aleexender** [comment]
성수기 비성수기를 따로 테이블로 만들어서 추후에 mySQL 스케쥴링 도입해서 날짜 지난 데이터들을 삭제하면 괜찮을거같아요!

**Comment by wocjf0513** [comment]
고민이 되긴 하는데, 저는!! 후자의 방법이 3개월 간의 예약 상품 데이터를 생성하면서 임의로 가격을 지정하는 거면 사실 후자의 방법을 택해서 서비스 상 크게 얻는 효과가 없을 거 같아서 전자가 좋을 거 같습니다!!

**Comment by Kwonminwoo** [comment]
저희 도메인에서 중요한 부분은 아닌것 같아서 개발 공수가 적은 성수기, 비성수기 구분 방법이 좋을 것 같습니다!
